examples/fineFoods/admin/antd/src/pages/orders/list.tsx

```diff
@@ -308,7 +283,7 @@ const Filter: React.FC<{ formProps: FormProps; filters: CrudFilters }> = (
                 <Col xl={24} md={8} sm={12} xs={24}>
                     <Form.Item
                         label={t("orders.filter.status.label")}
-                        name="status"
+                        name="status.text"
```

Comment:
```suggestion
                        name="status"
```
This is not working because `onSearch()` still uses `status`

```
export interface IOrderFilterVariables {
    q?: string;
    store?: string;
    user?: string;
    createdAt?: [Dayjs, Dayjs];
    status?: string;
}
```

```
filters.push({
                field: "status.text",
                operator: "in",
                value: status,
});
```

Reply:
@alicanerdurmaz

Is there any reason of using field name like "store", "user", "status" instead of the real name "store.id", "user.id", "status.text" beside just alias for easy name usage ?

The purpose of this PR is to avoid mapping for initial values when using form like the previous code handling here:

<img width="1072" alt="Screenshot 2022-11-12 at 10 57 49" src="https://user-images.githubusercontent.com/18494222/201456085-1c0d0b41-8bb1-4182-980a-f6be09d41fdc.png">

Reverting to field name "store", "user", "status" will bring back the mapping again and in terms of an example, it would be great to avoid confusion for developers by keep the default initial values (I've seen many cases that keep copy and paste the initial values mapping even that the field name is not nested field property).

In conclusion, just update "onSearch" callback is good enough:

```
onSearch: (params) => {

  filters.push({ field: 'status.text', value: params['status.text']  })
}

```

Reply:
Hi again @leapful, Sorry for late response.
Did you test the app with these code ? It's not filtering.

Because incoming params values are these.
`{
    "status.text": [
        "Ready"
    ],
    "store.id": 2,
    "user.id": 1
}`

But you are are trying to push with value `user` not `user.id`;
```
filters.push({
                field: "user.id",
                operator: "eq",
                value: user,
            });
```

by the way, if you wan to change variable name, you need to update [interface IOrderFilterVariables](https://github.com/refinedev/refine/blob/next/examples/fineFoods/admin/antd/src/interfaces/index.d.ts) too.

Reply:
yes this works. but you need to update the interface as well.

`onSearch: (params) => {
  filters.push({ field: 'status.text', value: params['status.text']  })
}
`

Reply:
@alicanerdurmaz 

Thanks for your review. I've pushed the change and already tested successfully in local development. Please help to review it again.